src/services/postApi.ts

```diff
@@ -0,0 +1,37 @@
+import axios from 'axios';
+
+const API_BASE_URL = 'http://localhost:8080/api/post';
```

Comment:
The API_BASE_URL is already configured in the .env and `/services/api.ts` You can refactor this service layer to be configured like the authService.ts